client/src/components/navbar/index.js

```diff
@@ -13,17 +13,30 @@ const NavBar = ({actualPath}) => (
   <nav className="navbar navbar-default">
     <div className="container-fluid">
       <div className="navbar-header">
+        <button
+          type="button"
+          className="navbar-toggle collapsed"
+          data-toggle="collapse"
+          data-target="#navbar"
```

Comment:
Done the last three. I've been lazy with this. I have not been able to get the animation of the NavBar. I leave it as a pending task. Maybe you could use [react-collapse](https://github.com/nkbt/react-collapse).